analyzer/src/funTest/assets/projects/external/git-repo-expected-output.yml

```diff
@@ -4095,18 +4095,26 @@ analyzer:
       NPM::isarray:2.0.5:
       - timestamp: "1970-01-01T00:00:00Z"
         source: "NPM"
-        message: "The package-lock.json file was created with an old version of npm,\n\
-          so supplemental metadata must be fetched from the registry.\nThis is a one-time\
-          \ fix-up, please be patient...\nminimatch@0.3.0: Please update to minimatch\
-          \ 3.0.2 or higher to avoid a RegExp DoS issue"
+        message: "The package-lock.json file was created with an old version of npm,\
```

Comment:
commit-msg: Maybe the last sentence could explicitly state that this fixes flakyness?!

Reply:
I've split out the fix for the flakyness of `BabelFunTest` to a separate commit.